CMSgov/easi-app
# [EASI-3088] IT Gov v2 feedback page

# EASI-3088

## Changes and Description

- Created [new feedback page](https://github.com/CMSgov/easi-app/blob/EASI-3088/feedback-edits-requested-page/src/views/GovernanceTaskList/Feedback/index.tsx) for IT Gov v2
  - Updated routes to hide new page behind feature flag
- Download feedback as PDF button functionality 
- Dynamic breadcrumbs and return link based on if you are navigating from the task list vs form feedback banner
  - Set using [location state](https://github.com/CMSgov/easi-app/blob/EASI-3088/feedback-edits-requested-page/src/views/GovernanceTaskList/Feedback/index.tsx#L35-L40) from [FeedbackBanner link](https://github.com/CMSgov/easi-app/blob/EASI-3088/feedback-edits-requested-page/src/components/FeedbackBanner/index.tsx#L39-L45)
- Unit tests  



## How to test this change

1. Go to /governance-review-team/:systemId/actions
2. Complete "Progress to new step" action (must have completed intake form for this action to work)
    - Fill out both "feedback for the requester" and "recommendations for the GRB" optional fields
3. Return to /governance-review-team/:systemId/actions
4. Complete "Request edits to a form" action
    - Select "Initial request form" for which form needs edits question
5. Go to feedback page - (/governance-task-list/:systemId/feedback)
6. Confirm feedback page works as expected and matches Figma
7. Go to intake form - (/system/:systemId/contact-details)
8. Click "View feedback" button on requested edits banner
9. Confirm that feedback page breadcrumbs and return links reference Intake Request Form



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
